stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -543,6 +544,68 @@ public Token createPiiTokenSynchronous(@NonNull String personalId, String publis
                 mLoggingResponseListener);
     }
 
+    /**
+     * Blocking method to create a {@link Token} for a Connect Account. Do not call this on the UI
+     * thread or your app will crash. The method uses the currently set
+     * {@link #mDefaultPublishableKey}.
+     *
+     * @param accountParams params to use for this token.
+     * @return a {@link Token} that can be used for this account.
+     *
+     * @throws AuthenticationException failure to properly authenticate yourself (check your key)
+     * @throws InvalidRequestException your request has invalid parameters
+     * @throws APIConnectionException failure to connect to Stripe's API
+     * @throws APIException any other type of problem (for instance, a temporary issue with
+     * Stripe's servers)
+     */
+    public Token createAccountTokenSynchronous(@NonNull final AccountParams accountParams)
+            throws AuthenticationException,
+            InvalidRequestException,
+            APIConnectionException,
+            CardException,
```

Comment:
The method signature also says this throws `CardException`, but it's not covered by the documentation block. The `createPii` methods have the same problem. Maybe just a copy/paste error from `createToken`?

Reply:
I made it deeper into the code, looks like this is because `StripeAPIHandler.createToken()` can throw `CardException`. For PII & Connect Account creation, I think you should probably "catch & swallow" `CardException` (**only** if it **cannot** happen, obviously). This prevents the underlying implementation (that you're sharing a method) from leaking to clients of the SDK. Otherwise I think it'd be good to document under what conditions it'd be thrown.

Reply:
Also, you might want to update variable names, etc, in `StripeAPIHandler.createToken()` - things like `cardParams` are now more general than just cards.